src/Components/ClickAndEdit.js

```diff
@@ -16,7 +20,14 @@ export default function ClickAndEdit({
   const theme = useTheme();
 
   const [editDesc, setEditDesc] = useState(false);
-  const [editedDesc, setEditedDesc] = useState(data);
+  const [editedDesc, setEditedDesc] = useState(undefined);
+
+  useEffect(() => {
```

Comment:
I have no idea why useState(data) worked for populating editedDesc for the "bio" section, but doing the same thing for "display Name" always gave me undefined....😠

Anyways, this was my workaround.

Reply:
Nice, this definitely seems like it will work.  With `useState` the initial value is trapped on the first evaluation, and must are first loading this component before the user profile is loaded in, so `data={profile?.name}` evaluates to undefined. 

Bio worked fine because we don't render the main profile page while the profile data is still loading.  See:
```
  if (isLoading) {
    return <ProfileMainPageGhost />;
  }
```
in `ProfileMainPage`.

By contrast, `ProfileUserBanner` doesn't have any such alternate ghost-display that is shown while the data deps are still loading.  If we add that, we wouldn't need to do this type of move bc `ClickAndEdit` would only be created once the data deps are present.  It might be a good thing to do anyways, since the 'loading' sstate in ProfileUserBanner doesn't quite match the ghost cards for the rest of the profile page.  Maybe leave a todo here to remove this trick after `ProfileUserBanner` has a ghost display implemented.